info.yml

```diff
@@ -14,7 +14,7 @@
 configuration:
     display_name:
         type: str
-        default_value: Send for review
+        default_value: ""
```

Comment:
It feels like this should be a required parameter instead of being skipped if there is no default value.